cli/wallet/wallet.go

```diff
@@ -563,12 +563,15 @@ func createWallet(ctx *cli.Context) error {
 }
 
 func readAccountInfo(w io.Writer) (string, string, error) {
-	rawName, _ := input.ReadLine(w, "Enter the name of the account > ")
-	phrase, err := input.ReadPassword(w, "Enter passphrase > ")
+	rawName, err := input.ReadLine("Enter the name of the account > ")
```

Comment:
For some reason if I'm to enter `2983$%#@.mdvgksdjgдлыоваъжх\` it not just fails, it also leaves terminal in unusable state (so that I have to use `reset` to restore it).

Reply:
This can be reproduced with simple `Ctrl+C` instead of account name. This is related to `restore` being done in `defer` which is not executed on `os.Exit`. For some reason this (unusable terminal state) only happens in Bash and not in Zsh or Fish. Moved all terminal handling to `input` package.